README.md

```diff
@@ -203,6 +203,13 @@ $ npm version major # for deprecation
 
 **Important:** `microplan` uses an automated publishing system. That means whenever your PR is merged, `microplan` will be published automatically to npm (based on the version specified in the package.json)
 
+## Contributors
+Authored and maintained by [@scriptnull](https://github.com/scriptnull) and [@argonlaser](https://github.com/argonlaser) with help from [contributors].
+
+[@scriptnull]: (https://github.com/scriptnull)
+[@argonlaser]: (https://github.com/argonlaser)
+[contributors]: http://github.com/microplan-xyz/microplan/contributors
```

Comment:
Let's remove this line and move the URL to line number 207 just like the usernames.